plugins/backfill/transformerExporter.go

```diff
@@ -0,0 +1,52 @@
+// This is a plugin generated to export the configured transformer initializers
+
+package main
+
+import (
+	deny "github.com/makerdao/vdb-mcd-transformers/transformers/events/auth/deny_initializer"
```

Comment:
I think that these events are ones that we've explicitly identified as ones that need to be backfilled - is the idea to avoid backfilling all of the events, because it would likely be too time consuming?

Reply:
Yeah, basically. I don't know if log fetching performance degrades when you query for more addresses/topics, but I think minimizing the result set we get back will help avoid attempted inserts to `public.event_logs` that are bound to fail on uniqueness constraint violations. Also, eventually I'm hoping that the set of transformers we're including here would only target very recently deployed contracts, so that there'd be less work to do scanning deployment blocks for contracts that were deployed a long time ago.